code/game/turfs/turf.dm

```diff
@@ -15,6 +15,10 @@
 	 * - an atmosphere id (use defines please)
 	 */
 	var/initial_gas_mix = GAS_STRING_TURF_DEFAULT
+	/**
+	 * Act like a specific temperature for heat exchanger pipes.
+	 */
+	var/temperature_for_heat_exchangers
```

Comment:
why are we moving this to /turf instead of /turf/simulated ?
The code that uses this var is only ever run on /turf/simulated

Reply:
so we can access it via `turf` level
it's not really about when it's used, it's about the fact that this is an inherent override to atmos behavior
everything else gets air from turf via API like return_air
there's no reason to have this one snowflake thing require `/simulated`; it's just against the usual atmos API to do so
a machine should not have knowledge of how the simulation for its turf is done, it should just do its job with the turf API

at some point we'll have `air_status` on `/turf` instead of `/turf/simulated` and `/turf/unsimulated`

after we evaluate what other optimizations unsim'd turfs have that simulated turfs don't anyways